src/gatling/simulations/simulations/ET_Simulation.scala

```diff
@@ -48,14 +62,14 @@ class ET_Simulation extends Simulation {
 
   val ETCreateClaim = scenario( "ETCreateClaim")
     .exitBlockOnFail {
-    //  .repeat(1){
+      //  .repeat(1){
       exec(  _.set("env", s"${env}"))
-      .exec(flushHttpCache)
-      .exec(flushCookieJar)
-      .feed(UserFeederET)
+        .exec(flushHttpCache)
+        .exec(flushCookieJar)
+        .feed(UserFeederET)
         .repeat(1) {
```

Comment:
Is this loop necessary?

Reply:
Not really, was used during the script making process. I have taken it out